br/pkg/utils/retry.go

```diff
@@ -104,6 +104,33 @@ func IsRetryableError(err error) bool {
 	return true
 }
 
+// IsRecoverableError attempts to catch network issue especially timeout error
+// in order to resume lightning automatically
```

Comment:
If this method is used by Lightning only, I think we can move this method to Lightning's common package.